network/api/v1/api.go

```diff
@@ -124,6 +124,32 @@ func (a Wrapper) RenderGraph(ctx echo.Context, params RenderGraphParams) error {
 	return ctx.String(http.StatusOK, visitor.Render())
 }
 
+func (a Wrapper) ListEvents(ctx echo.Context) error {
+	response := make([]EventSubscriber, 0)
+	for _, notifier := range a.Service.Subscribers() {
+		eventSubscriber := EventSubscriber{
+			Name: notifier.Name(),
+		}
+		events, err := notifier.GetFailedEvents()
+		if err != nil {
+			return err
+		}
+		for _, event := range events {
+			eventError := event.Error
```

Comment:
`len(event.Error) == 0` should be passed as `nil` error. Why is eventType optional?

Reply:
event type already was optional, so left it.